projects/plugins/jetpack/modules/sharedaddy/sharedaddy.php

```diff
@@ -118,7 +118,8 @@ function sharing_meta_box_save( $post_id ) {
 
 	// Record sharing disable.
 	if (
-		$post_type_object->public
+		$post_type_object instanceof WP_Post_Type
```

Comment:
Although this file doesn't use namespaces, should we be safer in the future by using this?
```suggestion
		$post_type_object instanceof \WP_Post_Type
```

Because `instanceof` won't fail if it can't find the class.

Reply:
Good suggestion.